luigi_pipeline/seqr_gcnv_loading.py

```diff
@@ -36,7 +60,7 @@ def generate_callstats(self, mt, *args, **kwargs):
         return mt
 
     def import_dataset(self):
-        ht = hl.import_table(self.source_paths[0], impute=True, min_partitions=500)
+        ht = hl.import_table(self.source_paths[0], types=EXPECTED_TYPES, min_partitions=500)
```

Comment:
Validation happens lazily here, which I think is fine?  There's a very clear error message:

```
HailUserError: Error summary: HailException: cannot parse boolean from input string 't'
```

Reply:
most things in hail are lazy, so thats fine with me